indico/web/templates/forms/_person_link_widget_base.html

```diff
@@ -1,35 +1,23 @@
 {% extends 'forms/base_widget.html' %}
 
+{% macro render_section(id='other', title=none, placeholder=none) %}
+    {% if title is not none %}
+        <div id="{{ id }}-list-title-{{ field.id }}" class="titled-rule">
```

Comment:
I would pass the field as an argument to the macro. Using global data inside a macro is a bit strange in my opinion